lib/adiwg/mdtranslator/readers/iso19115_2/modules/module_temporal_extent.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'nokogiri'
+require 'adiwg/mdtranslator/internal/internal_metadata_obj'
+require_relative 'module_time_period'
+
+module ADIWG
+   module Mdtranslator
+      module Readers
+         module Iso191152
+            module TemporalExtent
+               @@temporalExtXPath = 'gmd:EX_TemporalExtent'
+               @@timePeriodXPath = './/gml:TimePeriod'
+               def self.unpack(xtemporalelem, hResponseObj)
+                  intMetadataClass = InternalMetadata.new
+                  hTemporalExt = intMetadataClass.newTemporalExtent
+
+                  xTemporalExt = xtemporalelem.xpath(@@temporalExtXPath)[0]
+                  return nil if xTemporalExt.nil?
+
+                  # :timePeriod
+                  # <sequence minOccurs="0"> <element ref="gml:TimePeriod"/> </sequence>
+                  xTimePeriod = xTemporalExt.xpath(@@timePeriodXPath)[0]
+                  hTemporalExt[:timePeriod] = TimePeriod.unpack(xTimePeriod, hResponseObj) unless xTimePeriod.nil?
```

Comment:
`TimePeriod.unpack` should be given `gmd:extent` not `gml:TimePeriod`.

Reply:
every module is responsible for finding itself. in this case, `module_time_period` is expected to find `gml:TimePeriod`.